app/uk/gov/hmrc/registerncfstub/controllers/RegisterNcfController.scala

```diff
@@ -42,28 +42,28 @@ class RegisterNcfController @Inject()(appConfig: AppConfig, registerNcfService:
             responseWithCorrelationIdHeader(Ok(Json.toJson(NcfResponse(mrn, responseCode, None))))
           case TechnicalError(mrn, responseCode, e) =>
             logBadRequests(mrn, responseCode, e)
-            responseWithCorrelationIdHeader(BadRequest(Json.toJson(NcfResponse(mrn, responseCode, Some(e)))))
+            responseWithCorrelationIdHeader(Ok(Json.toJson(NcfResponse(mrn, responseCode, Some(e)))))
           case ParsingError(mrn, responseCode, e) =>
             logBadRequests(mrn, responseCode, e)
```

Comment:
Can we rename this to 'logResponse' as it's no longer a 400 so doesn't  really make sense being the current name